### PR TITLE
OWLS-73314, added tests for monitoring exporter

### PR DESCRIPTION
**Pull request by marinakog**

### CODE REVIEW

**Comment by vanajamukkara** [comment]
@marinakog I am not familiar with monitoring exporter, please get the use cases and code reviewed by someone who is familiar. Also, can you make sure these tests would work on remote multi domain k8s cluster as well and update USECASES.md.

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java [hunk 1]
```diff
@@ -928,6 +934,155 @@ public static void createDirUnderDomainPV(String dirPath) throws Exception {
...
+    }
+  }
+
+  /**
+   * Remove monitoring exporter directory if exists and clone latest from github for monitoring
+   * exporter code
+   *
+   * @throws Exception if could not run the command successfully to clone from github
+   */
+  public static void gitCloneBuildMonitoringExporter() throws Exception {
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: If these methods are used only in Monitoring Exporter testing, you can move them to the IT class. Also please add more comments inside the code.

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java [hunk 2]
```diff
@@ -1337,4 +1492,19 @@ public static void buildDeployWebServiceAppInPod(
     domain.callShellScriptToBuildDeployAppInPod(
         appName, scriptName, username, password, clusterURL, wsServiceName);
   }
+
+  public static void executeCmd(String cmd) throws Exception {
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: can you see if you can use exec(str) method in TestUtils.

Comment (marinakog) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: changed to exec

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: moved all to ITMonitoringExporter

**Comment by marinakog** [comment]
all tests passed in http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/1632/

except this one : 
 ITOperator.testDomainInImageUsingWLST:454 ? Runtime FAILURE: pod domaininimage.

it consistently fail in all runs 1625-1633

**Pull request merged**